user/logging.go

```diff
@@ -7,20 +7,21 @@ package user
 import (
 	"context"
 
-	"github.com/grafana/dskit/log"
+	"github.com/go-kit/log"
 )
 
 // LogWith returns user and org information from the context as log fields.
-func LogWith(ctx context.Context, log log.Interface) log.Interface {
+func LogWith(ctx context.Context, logger log.Logger) log.Logger {
 	userID, err := ExtractUserID(ctx)
+	logergWithFields := logger
```

Comment:
[nit] There's a typo "logerg" instead of "logger", but I don't think you need this variable at all. You can just overwrite `logger`.